Tools/CISupport/ews-build/config.json

```diff
@@ -350,7 +384,13 @@
       "name": "API-Tests-macOS-EWS", "shortname": "api-mac", "icon": "testOnly",
       "factory": "APITestsFactory", "platform": "*",
       "triggered_by": ["macos-monterey-release-build-ews"],
-      "workernames": ["ews100", "ews129", "ews142", "ews143", "ews150", "ews153", "ews155"]
+      "workernames": ["ews100", "ews129", "ews142", "ews143", "ews150"]
+    },
+    {
+      "name": "API-Tests-macOS-AppleSilicon-Ventura-EWS", "shortname": "api-ventura-mac", "icon": "testOnly",
```

Comment:
Now that most/all mac queues will be on AppleSilicon, we should consider dropping the AppleSilicon from queue names and adding intel/x86 wherever appropriate.